build/package/Microsoft.DotNet.Cli.Installer.DEB.targets

```diff
@@ -1,9 +1,114 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
     
-    <UsingTask TaskName="DebTargets" AssemblyFile="$(CLIBuildDll)" />
+    <Target Name="SetupGenerateDebInputsOutputs">
+        <Exec Command="/usr/bin/env debuild -h" ContinueOnError="true">
+            <Output TaskParameter="ExitCode" PropertyName="DebuildExitCode" />
+        </Exec>
 
-    <Target Name="GenerateDebs" >
-      <DebTargets CLISDKRoot="$(LayoutDirectory)/$(ArtifactNameSdk)/sdk" />
+        <PropertyGroup>
+            <DebuildPresent>false</DebuildPresent>
+            <DebuildPresent Condition=" '$(DebuildExitCode)' == '0' ">true</DebuildPresent>
+
+            <DebianPackagingScript>$(RepoRoot)/scripts/package/package-debian.sh</DebianPackagingScript>
+            <ManpagesDirectory>$(RepoRoot)/Documentation/manpages</ManpagesDirectory>
+            
+            <InstallerOutputDirectory>$(PackagesDirectory)</InstallerOutputDirectory>
+            <SdkInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionSdk)$(InstallerExtension)</SdkInstallerFile>
+        
+            <SdkDebianPackageName>dotnet-dev-$(SdkVersion)</SdkDebianPackageName>
+            <SharedFrameworkDebianPackageName>dotnet-sharedframework-$(SharedFrameworkName)-$(SharedFrameworkVersion)</SharedFrameworkDebianPackageName>
+            <SharedFrameworkDebianPackageName>$(SharedFrameworkDebianPackageName.ToLower())</SharedFrameworkDebianPackageName>
+            <HostFxrDebianPackageName>dotnet-hostfxr-$(HostFxrVersion)</HostFxrDebianPackageName>
+            <HostFxrDebianPackageName>$(HostFxrDebianPackageName.ToLower())</HostFxrDebianPackageName>
+            <SharedHostDebianPackageName>dotnet-host</SharedHostDebianPackageName>
+
+            <SdkDebianIntermediateDirectory>$(IntermediateDirectory)/debian/sdk</SdkDebianIntermediateDirectory>
+        
+            <DebianTestResultsXmlFile>$(SdkDebianIntermediateDirectory)/debian-testResults.xml</DebianTestResultsXmlFile>
+            <DebianInstalledDotnet>/usr/share/dotnet/dotnet</DebianInstalledDotnet>
+            <EndToEndTestDirectory>$(RepoRoot)/test/EndToEnd</EndToEndTestDirectory>
+
+            <SdkVersionSentinel>$(SdkDebianIntermediateDirectory)/$(SdkVersion).sentinel</SdkVersionSentinel>
+        </PropertyGroup>
+
+        <ItemGroup>
+            <SdkDebInputFiles Include="$(SdkLayoutOutputDirectory)/**/*" />
+        </ItemGroup>
+
+        <!-- Rebuild when version changes -->
+        <MakeDir Directories="$(SdkDebianIntermediateDirectory)" />
+        <WriteLinesToFile
+          File="$(SdkVersionSentinel)"
+          Lines="$(SdkVersion)"
+          Condition= "!Exists('$(SdkVersionSentinel)')"
+          Encoding="Unicode"/>
+
+        <Message Condition=" '$(DebuildPresent)'  != 'true' " 
```

Comment:
The Importance high was intended to signify a warning, but no it shouldn't be an error. I'll add back the comment that htis is a workaround for ubuntu 16 not present on jenkins